### ISSUE TITLE
How to register or login on programmer? There is not any way to do that?

### ISSUE THREAD

**Issue by MEGADRAGSTER288** [MineMaarten/PneumaticCraft#892]
hi MineMarteen, i want to login or register to the programmer but i dont know how! can you explain how it works? I want to login on it but i dont know how!! If you can explain i will be very happy! But i need to have the authenticated account on Minecraft? Or it is not needed? Cause i not have it

**Comment by Forecaster** [comment]
What are you talking about. The programmer doesn't require logging in for anything other than the pastebin integration.

**Comment by MEGADRAGSTER288** [comment]
Yes but what is that login and password on the programmer when you click to make your pastebin?

**Comment by Forecaster** [comment]
For pastebin.
